config/master/aws-k8s-cni.yaml

```diff
@@ -70,6 +70,10 @@ rules:
     resources:
       - '*'
     verbs: ["list", "watch"]
+  - apiGroups: ["", "events.k8s.io"]
```

Comment:
Let's update the remaining yaml i.e, cn, gov regions. `make generate-cni-yaml` should generate all yamls in build folder.